includes/TwigEnvironment.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Twig Environment for GbWeiss Plugin
+ *
+ * @package GbWeissTwig
+ */
+
+namespace GbWeiss\includes;
+
+defined('ABSPATH') || exit;
+
+use Exception;
+use Twig\Environment;
+use Twig\Extension\DebugExtension;
+use Twig\Loader\FilesystemLoader;
+use Twig\TwigFunction;
+
+/**
+ * Twig Environment Singleton
+ */
+class TwigEnvironment extends Singleton
+{
+    /**
+     * Twig Loader
+     *
+     * @var FilesystemLoader
+     */
+    private $loader;
+
+    /**
+     * Twig Environment
+     *
+     * @var Environment
+     */
+    private $twig;
+
+    /**
+     * Instatiate new Twig Environment
+     */
+    public function __construct()
+    {
+        $this->loader = new FilesystemLoader(__DIR__ . '/../templates/');
+        $this->twig = new Environment($this->loader, [
+        'debug' => true
+        ]);
+        $this->addTwigExtensions();
+    }
+
+    /**
+     * Prevent the instance from being cloned (which would create a second instance of it)
+     */
+    protected function __clone()
+    {
```

Comment:
should this throw an exception?

Reply:
no, it prevens someone using it from doing so.

Reply:
i understand :)